app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user, optional: true
+  has_many :comments
+  has_many :likes
+
+  after_save :update_post_counter
+
+  def recent_comments
+    Comment.where(post_id: id).order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] You can leverage on the associations by using this 👇🏾  method instead. I have explained why in my previous comment on a similar method. 

```ruby
def recent_comments
  comments.order('created_at Desc').limit(5)
end
```

Reply:
Fixed.